Items Properties/Nantho_item Auto Gain Staging - Peak vs. LUFS.lua

```diff
@@ -0,0 +1,169 @@
+--[[
+ * Provides: [nomain] spk77_Get max peak val and pos from take_function.lua > Nantho_Item Auto-Gain Staging - Peak vs. RMS.lua/spk77_Get max peak val and pos from take_function_Copy2.lua
```

Comment:
This could go in a "Peak vs. LUFS" folder to match the script's name, rather than both Copy1+Copy2 in one "RMS" directory (and then suffixing with CopyX is no longer necessary).